src/main/resources/liquibase.properties

```diff
@@ -0,0 +1,4 @@
+url=jdbc:mysql://localhost:3306/book_shop
+username=root
+password=MySQL1234
```

Comment:
As I know, it's bad practice to pass the login and password to connect to the database in the configuration files.